### PR TITLE
feat(radio): add new component

### PR DESCRIPTION
**Pull request by yasminnvaz**
# Description

add new component

## Type of change

- [x] Feat: A new feature (non-breaking change which adds functionality)
- [ ] Refactor: A code change that neither fixes a bug nor adds a feature
- [ ] Breaking change: fix or feature that would cause existing functionality to not work as expected
- [ ] Docs: Documentation only changes

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings or errors;
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) Guidelines;

### CODE REVIEW

File: packages/react/src/components/Radio/Radio.props.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+   */
+  id?: string
+
+  /**
+   * Name attribute of the `input` element.
+   */
+  name?: string
+
+  /**
+  * The onchange event occurs when the checked state has been changed.
```
Comment (arielwb) [inline packages/react/src/components/Radio/Radio.props.ts:27]: `onChange`

File: packages/react/src/components/Radio/Radio.props.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  */
+  onChange: React.ChangeEventHandler<HTMLInputElement>,
+
+  /**
+   * Pass a ref to the `input` element.
+   */
+  ref?: React.Ref<never>
+
+  /**
+   * If `true`, the `input` element will be required.
```
Comment (arielwb) [inline packages/react/src/components/Radio/Radio.props.ts:37]: eu senti que a gente chamou o radio de component, Radio, e aqui de input.
Sera que vale padronizar?

File: packages/react/src/components/Radio/Radio.props.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+  /**
+   * If `true`, the Radio will be disabled.
+   * @default false
+   */
+  disabled?: boolean
+
+  /**
+   * The id of the Radio element.
+   * Must be the same as label
```
Comment (arielwb) [inline packages/react/src/components/Radio/Radio.props.ts:17]: Aqui eu fiquei na dúvida, e olhando o pr, eu senti falta de uma prop pra ser a label, ela já vir com a font padrão, espaçamento etc, e tbm com o id certinho.
Se a usuaria nao passa, se retorna so o radio. 
Acho que no RN é assim? 🤔 
faz sentido?

File: packages/react/src/components/Radio/Radio.styles.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  radio: {
+    appearance: 'none',
+    border: `2px solid ${theme.color.primary}`,
+    borderRadius: '50%',
+    cursor: 'pointer',
+    display: 'inline-block',
+    height: 18,
+    margin: 0,
+    position: 'relative',
+    width: 18,
```
Comment (lauralucca) [inline packages/react/src/components/Radio/Radio.styles.ts:23]: Será que aqui valeria a pena usar um token small ou algum cálculo de proporção com o semiX do wrapper?

File: packages/web/package.json [hunk 5]
```diff
@@ -2,7 +2,7 @@
   "name": "@naturacosmeticos/natds-web",
   "displayName": "NatDS React Web",
   "description": "A collection of components from Natura Design System for React websites and webapps",
-  "version": "4.0.1-alpha.DSY-2195.13.0",
+  "version": "4.0.1-alpha.DSY-1942.21.0",
```
Comment (lauralucca) [inline packages/web/package.json:5]: essas alterações no pacote web deviam acontecer?

File: packages/react/src/components/Radio/Radio.props.ts (hunk 3 above)
Comment (lauralucca) [reply packages/react/src/components/Radio/Radio.props.ts:17]: No RN é assim sim. Acho que faz sentido.

**Pull request merged**